### PR TITLE
Feat: (replat 7163) fix issues from design review

### PR DESCRIPTION
**Pull request by jmars**

### CODE REVIEW

File: packages/article-skeleton/src/article-skeleton.js [hunk 1]
```diff
@@ -30,7 +34,7 @@ const viewabilityConfig = {
 };
 
 const convertStyles = ({ font, size }) => ({
-  fontFamily: font,
+  fontFamily: "TimesDigitalW04",
```
Comment (parkjh) [inline packages/article-skeleton/src/article-skeleton.js]: why is it hard coded?

Comment (parkjh) [reply packages/article-skeleton/src/article-skeleton.js]: if you remove it or have it as default parameter value for font I am happy to approve it

File: packages/article-skeleton/src/article-body/article-body-row.js [hunk 2]
```diff
@@ -88,7 +88,9 @@ export default ({
       const cap = getDropCap(children, fonts[dropCapFont], height, [
         new Body(value)
       ]);
-      const capWidth = (cap[0].measuredWidth + 20) * fontScale;
+      const dropCapGap =
+        typeof value === "string" && ["M", "W"].includes(value) ? 18 : 10;
```
Comment (ValGeorgiev) [inline packages/article-skeleton/src/article-body/article-body-row.js]: What are these values here? 10 & 18

File: packages/caption/src/centred-caption.js [hunk 3]
```diff
@@ -10,6 +10,7 @@ const CentredCaption = ({ children, credits, style, text }) => (
       ...style,
       text: {
         ...style.text,
+        marginTop: 5,
```
Comment (ValGeorgiev) [inline packages/caption/src/centred-caption.js]: could we use spacing?

File: packages/caption/src/styles/index.android.js [hunk 4]
```diff
@@ -10,11 +10,13 @@ const styles = StyleSheet.create({
...
     ...sharedStyles.credits,
-    lineHeight: 20
+    lineHeight: 20,
+    marginTop: 0
   },
   text: {
     ...sharedStyles.text,
-    lineHeight: 20
+    lineHeight: 20,
+    marginTop: -5
```
Comment (ValGeorgiev) [inline packages/caption/src/styles/index.android.js]: spacing here as well

**Pull request merged**